### ISSUE TITLE
DNSMessageParser compile error

### ISSUE THREAD

**Issue by markheger** [IBMStreams/streamsx.network#192]
with #191  the following error is observed
```
    [exec]   [sc main] In file included from src/operator/DNSMessageStream.cpp:6:0:
     [exec]   [sc main] src/operator/./DNSMessageStream.h: In member function ‘SPL::rstring SPL::_Operator::DNSMessageStream$OP::DNS_ALL_FIELDS(SPL::rstring, SPL::rstring, SPL::rstring, SPL::list<short unsigned int>, SPL::uint64, SPL::uint32, SPL::uint32, SPL::uint32, SPL::uint16, SPL::uint16, SPL::uint8, SPL::uint32, SPL::uint16)’:
     [exec]   [sc main] src/operator/./DNSMessageStream.h:283:243: error: no matching function for call to ‘DNSPacketFlattener::dnsAllFields(SPL::uint64&, SPL::uint32&, unsigned int, unsigned int, SPL::uint16&, SPL::uint16&, SPL::uint8&, SPL::uint32&, SPL::uint16&, DNSMessageParser&, const char*, const char*, const char*, SPL::list<short unsigned int>&)’
     [exec]   [sc main]      { return flattener.dnsAllFields(captureSeconds, packetLen, htonl(srcAddr), htonl(dstAddr), srcUdpPort, dstUdpPort, protocol, payloadLength, mac16, parser, recordDelimiter.c_str(), fieldDelimiter.c_str(), subfieldDelimiter.c_str(), rrTypes); }
```

**Comment by jlkaus** [comment]
Sorry about that.  Was building with a different version of the flattener.  Will open a new pull request with the fix in a bit here, after testing (with the proper versions!)

**Comment by jlkaus** [comment]
Pull request #193 fixes this compilation error.

**Comment by markheger** [comment]
fixed in 3.2.1
